packages/client/src/services/DocsService.ts

```diff
@@ -55,8 +55,8 @@ export default class DocsService {
   async fetchAndUpdateToc(): Promise<void> {
     if (docsStore.tableOfContents == null) {
       try {
-        const response = await axios.get(`${process.env.PUBLIC_URL}/docs/table-of-contents.yaml`);
-        const tocData: any = yaml.safeLoad(response.data);
+        const data = await this.fetchToc();
+        const tocData: any = yaml.safeLoad(data);
```

Comment:
FYI, this might get done with the rebase from the dependency updates but safeLoad has been deprecated and we should just use yaml.load(data) now. It's safe by default in the newest version.

Reply:
Thanks for the heads up, I made sure we are using yaml.load(data) after the rebase